### DISCUSSION TITLE
Scheduled reminders does not see all projects

### DISCUSSION THREAD

**Discussion by Newbas** [community/community#23762] [New to GitHub]
<p>We set team with access to 6 repositories (3 forks from private repositories and 3 regular private). We set auto assignments for PR review and want to use reminders. But for some reason reminder see only 2 regular projects. It does not see forks, and in the begin i thought it is cause this issue. But it also does not see just created private repo (without fork). Checked setting and they looks same with 2 others.</p>
<p>Any reason why i can’t see all repos? <br>
Should it work with forked repos? </p>

**Comment by ghost** [comment, marked as answer]
<p>Hi <a class="mention" href="/newbas">@newbas</a> - thanks for reaching out on the GitHub forums.</p>
<p>You should be able to add scheduled reminders for a forked repository, as long as the repo has been assigned to the team. You would need to add the scheduled reminder for that specific team by going to the team settings page. (e.g. <a href="https://github.com/orgs/orgname/teams/teamname/settings/reminders/new">https://github.com/orgs/orgname/teams/teamname/settings/reminders/new</a>)</p>
<p>If a repo is not showing when adding reminders, check that the team has the correct permissions for that repo.</p>

**Comment by Newbas** [comment]
<p>Hello <a class="mention" href="/robertnorrie">@robertnorrie</a>,</p>
<p>thanks for your answer, issue is that team has 5 repositories with same permission (write) and i can see only one now if try to set reminders for specific repo. And in case i set all repos, i receive reminders only from same one repo. Not sure why it so (previously as i mentioned i saw 2 repos).</p>
<p>Thanks</p>

**Reply by nicollecnunes** [reply]
did you solve it?
